{
  "dataType": "CVE_RECORD",
  "containers": {
    "adp": [
      {
        "title": "CVE Program Container",
        "references": [
          {
            "url": "https://github.com/sddm/sddm/releases",
            "tags": [
              "x_transferred"
            ]
          },
          {
            "url": "https://bugzilla.suse.com/show_bug.cgi?id=CVE-2020-28049",
            "tags": [
              "x_transferred"
            ]
          },
          {
            "url": "https://github.com/sddm/sddm/blob/v0.19.0/ChangeLog",
            "tags": [
              "x_transferred"
            ]
          },
          {
            "url": "https://www.debian.org/security/2020/dsa-4783",
            "name": "DSA-4783",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          },
          {
            "url": "https://lists.debian.org/debian-lts-announce/2020/11/msg00009.html",
            "name": "[debian-lts-announce] 20201106 [SECURITY] [DLA 2436-1] sddm security update",
            "tags": [
              "mailing-list",
              "x_transferred"
            ]
          },
          {
            "url": "http://lists.opensuse.org/opensuse-security-announce/2020-11/msg00031.html",
            "name": "openSUSE-SU-2020:1870",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          },
          {
            "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/GT3EX5NSQJJAKY63ENSMEDX6NYZLYY3S/",
            "name": "FEDORA-2021-7066b95c99",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          },
          {
            "url": "https://security.gentoo.org/glsa/202402-02",
            "name": "GLSA-202402-02",
            "tags": [
              "vendor-advisory",
              "x_transferred"
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "af854a3a-2127-422b-91ae-364da2661108",
          "shortName": "CVE",
          "dateUpdated": "2024-08-04T16:33:56.915Z"
        }
      },
      {
        "title": "CISA ADP Vulnrichment",
        "metrics": [
          {
            "cvssV3_1": {
              "scope": "UNCHANGED",
              "version": "3.1",
              "baseScore": 6.3,
              "attackVector": "LOCAL",
              "baseSeverity": "MEDIUM",
              "vectorString": "CVSS:3.1/AV:L/AC:H/PR:L/UI:N/S:U/C:H/I:H/A:N",
              "integrityImpact": "HIGH",
              "userInteraction": "NONE",
              "attackComplexity": "HIGH",
              "availabilityImpact": "NONE",
              "privilegesRequired": "LOW",
              "confidentialityImpact": "HIGH"
            }
          },
          {
            "other": {
              "type": "ssvc",
              "content": {
                "id": "CVE-2020-28049",
                "role": "CISA Coordinator",
                "options": [
                  {
                    "Exploitation": "none"
                  },
                  {
                    "Automatable": "no"
                  },
                  {
                    "Technical Impact": "total"
                  }
                ],
                "version": "2.0.3",
                "timestamp": "2024-10-15T17:16:10.736062Z"
              }
            }
          }
        ],
        "problemTypes": [
          {
            "descriptions": [
              {
                "lang": "en",
                "type": "CWE",
                "cweId": "CWE-362",
                "description": "CWE-362 Concurrent Execution using Shared Resource with Improper Synchronization ('Race Condition')"
              }
            ]
          }
        ],
        "providerMetadata": {
          "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
          "shortName": "CISA-ADP",
          "dateUpdated": "2024-10-15T18:10:18.118Z"
        }
      }
    ],
    "cna": {
      "affected": [
        {
          "vendor": "n/a",
          "product": "n/a",
          "versions": [
            {
              "status": "affected",
              "version": "n/a"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://github.com/sddm/sddm/releases"
        },
        {
          "url": "https://bugzilla.suse.com/show_bug.cgi?id=CVE-2020-28049"
        },
        {
          "url": "https://github.com/sddm/sddm/blob/v0.19.0/ChangeLog"
        },
        {
          "url": "https://www.debian.org/security/2020/dsa-4783",
          "name": "DSA-4783",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://lists.debian.org/debian-lts-announce/2020/11/msg00009.html",
          "name": "[debian-lts-announce] 20201106 [SECURITY] [DLA 2436-1] sddm security update",
          "tags": [
            "mailing-list"
          ]
        },
        {
          "url": "http://lists.opensuse.org/opensuse-security-announce/2020-11/msg00031.html",
          "name": "openSUSE-SU-2020:1870",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/GT3EX5NSQJJAKY63ENSMEDX6NYZLYY3S/",
          "name": "FEDORA-2021-7066b95c99",
          "tags": [
            "vendor-advisory"
          ]
        },
        {
          "url": "https://security.gentoo.org/glsa/202402-02",
          "name": "GLSA-202402-02",
          "tags": [
            "vendor-advisory"
          ]
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "value": "An issue was discovered in SDDM before 0.19.0. It incorrectly starts the X server in a way that - for a short time period - allows local unprivileged users to create a connection to the X server without providing proper authentication. A local attacker can thus access X server display contents and, for example, intercept keystrokes or access the clipboard. This is caused by a race condition during Xauthority file creation."
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "text",
              "description": "n/a"
            }
          ]
        }
      ],
      "providerMetadata": {
        "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "shortName": "mitre",
        "dateUpdated": "2024-02-03T07:06:30.199645"
      }
    }
  },
  "cveMetadata": {
    "cveId": "CVE-2020-28049",
    "state": "PUBLISHED",
    "dateUpdated": "2024-10-15T18:10:25.839Z",
    "dateReserved": "2020-11-02T00:00:00",
    "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
    "datePublished": "2020-11-04T00:00:00",
    "assignerShortName": "mitre"
  },
  "dataVersion": "5.1"
}